src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Calculator.css';
+// eslint-disable-next-line
```

Comment:
- [ ] Please remove the `eslint-disable` line and try to fix it manually as this defies the purpose of using linters in the first place.

Reply:
Yes, I understand, but the linters are forcing me to change the class base component to pure function, and when I submitted it as function the reviewer told me to change it to class base component. Here is the linter error:
![image](https://user-images.githubusercontent.com/114300121/211834117-9eaefa44-e090-4ee9-a142-87846548179c.png)